R/carehomes_ifr_t.R

```diff
@@ -7,8 +7,8 @@
 ##' @param S A (n groups x n vaccine classes) x steps matrix of
 ##'   "S" compartment counts
 ##'
-##' @param I_weighted A (n groups x n vaccine classes) x steps matrix of
-##'   "I_weighted" compartment counts
+##' @param I_weighted A (n groups x n strains x n vaccine classes) x steps
```

Comment:
looks like this is still accepted without the strain dimension? Also change matrix -> array

Reply:
It's still a matrix, before the 1st dim was of length (n groups x n vaccine class), now it is (n groups x n strains x n vaccine classes)